src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -51,30 +54,42 @@
                                          {:animated true
                                           :offset   0})))
                    [(count cards)])
+    (rn/use-effect
+     #(when (and (boolean? tokens-loading?) (not tokens-loading?) (not init-loaded?))
+        (set-init-loaded true))
+     [tokens-loading?])
     [rn/view {:style (style/home-container)}
      [common.top-nav/view]
-     [rn/view {:style style/overview-container}
-      [quo/wallet-overview
-       {:state             (if tokens-loading? :loading :default)
-        :time-frame        :none
-        :metrics           :none
-        :balance           formatted-balance
-        :networks          networks
-        :dropdown-on-press #(rf/dispatch [:show-bottom-sheet {:content network-filter/view}])}]]
-     [quo/wallet-graph {:time-frame :empty}]
-     [rn/flat-list
-      {:ref                               #(reset! account-list-ref %)
-       :style                             style/accounts-list
-       :content-container-style           style/accounts-list-container
-       :data                              cards
-       :horizontal                        true
-       :separator                         [rn/view {:style style/separator}]
-       :render-fn                         (fn [item] [quo/account-card item])
-       :shows-horizontal-scroll-indicator false}]
-     [quo/tabs
-      {:style          style/tabs
-       :size           32
-       :default-active selected-tab
-       :data           tabs-data
-       :on-change      #(set-selected-tab %)}]
-     [tabs/view {:selected-tab selected-tab}]]))
+     [rn/scroll-view
+      {:refresh-control (reagent/as-element
```

Comment:
I wonder can we abstract this as-element away?

Reply:
I removed it but I got errors. @ulisesmac  added this and I kept it.

Reply:
Im not saying to remove it, I mean to abstract it away so we don't need to add this method each time we use the pull to refresh,  there is many other uses of pull to refresh in the designs so it's nice to take steps like this

Reply:
Oh, right. We can create a new refresh-control component and add this there

Reply:
nice yeah that sounds good to me!